package.json

```diff
@@ -23,14 +23,16 @@
     "storybook": "start-storybook -p 6006",
     "storybook:build": "build-storybook -c .storybook -o public/storybook",
     "test": "jest",
-    "ts:check": "tsc --noEmit"
+    "ts:check": "tsc --noEmit",
+    "import": "npx ts-node lib/js/importers/RawColorsImporter.ts && npx ts-node lib/js/importers/TokensImporter.ts"
```

Comment:
these scripts aren't defined on this branch

Reply:
As I mentioned above, now is only one script, I will update this line.